src/js/components/Anchor/doc.js

```diff
@@ -59,7 +59,9 @@ or just use children.`,
 this component. But, it can be adjusted directly via this size property, typically
 when it is not contained in a 'Heading', 'Paragraph', or 'Text'.`,
     ),
-    as: PropTypes.string.description(`The DOM tag to use for the element.`),
+    as: PropTypes.oneOfType([PropTypes.string, PropTypes.func]).description(
+      `The DOM tag to use for the element.`,
```

Comment:
can we change the description to reflect this change as well?

Reply:
Ok in the end this is cumbersome to do since there are different descriptions (deprecation on not) and default values